router.go

```diff
@@ -203,16 +203,16 @@ func (r *Router) InterceptLinks() {
 		href := link.GetAttribute("href")
 		switch {
 		case href == "":
-			return
+
```

Comment:
Can you please add a `continue` statement in each instance where you removed a `return` statement for the purpose of clarity?